config/images.yml

```diff
@@ -143,6 +143,12 @@
   variants:
     - "media/images/content/hero-images/0024--mobile.jpg"
     - "media/images/content/hero-images/0024--tablet.jpg"
+    
+"media/images/content/hero-images/0025.jpg":
+  alt: "Helper talking to an attendee at an in person event."
```

Comment:
We should take the opportunity to maximise the use of key word searches so changing the image description. Perhaps "Teacher training provider advising a potential teacher trainee at an in person teacher training event"? think that is 101 characters so need to think about making that succinct.

Reply:
SEO definitely important as discussed - including 'teacher training event' definitely a good idea

However, do be mindful of screen readers which is the main purpose of alt text - this is very verbose/repetitive if you are listening to the description being read out?

Reply:
Yes makes sense - was an initial draft and suggestion and was hoping others could input/review before changes were made.

Reply:
@peteryates do we need to have a 'variant image' when adding the alt text? Not quite sure how this works

Reply:
No, the variant entry is only needed when there are multiple versions of the same photo. I've pushed @EllieNodder's alt text for the homepage images.